packages/facebook/configuration-schema.json

```diff
@@ -7,10 +7,15 @@
       "type": "string",
       "description": "Your Facebook API access token",
       "writeOnly": true,
-      "minLength": 1
+      "minLength": 1,
+      "examples": [
+        "EAAGm0PX4ZCpsBAEHuZBwZBZCDEZDZD7ZC8ZBZBt7MbZA7sVZA...."
```

Comment:
Two things:

1) the `...` is a bit weird, do we need it? Can we just leave a partial token? 

2) For the record, `...` is an ellipsis (from greek to leave out"). It always has _3_ docs, Not 2 dots, not 4 dots, not 27 dots. Always three.

Reply:
The actual token was very long, let me remove the dots, Thanks for sharing about ellipsis, I was totally miss using it 🤣

Reply:
I have removed the dot's from the token example

Reply:
Great thanks. I think it's fine to shorten it and we don't really need to advertise that fact. It's not like anyone is going to copy our key (and if they try, tough!)